### ISSUE TITLE
Standardize document and Issue Labels

### ISSUE THREAD

**Issue by A1Liu** [YACS-RCOS/docs#19]
TODO from docs/contributors/getting_started.md

**Comment by Bad-Science** [comment]
Thanks for opening this! This is actually pretty much done, I just haven't added it to the docs. See the `yacs` repo labels for more info.

**Comment by A1Liu** [comment]
Yeah, I copy-pasted them into the docs in #20 , I didn't add the html styling but I can probs add that without too much trouble
